packages/components/src/components/flipcard/flipcard.stories.ts

```diff
@@ -0,0 +1,275 @@
+
+  argTypes,
+
+  parameters: { ...parameters },
+  decorators: [withActions] as any
+};
+/**
+ * This shows sd-flipcard in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return html`<div style="width:300px">${generateTemplate({ args })}</div>`;
+  }
+};
+
+/**
+ * The sd-flipcard can be displayed in several ways using the `frontVariant` and `backVariant` attributes. This example shows the usage `frontVariant` attribute.
+ */
+
+export const Variants = {
+  parameters: { controls: { exclude: ['frontVariant'] } },
+  render: (args: any) =>
+    generateTemplate({
+      axis: {
+        y: {
+          type: 'attribute',
+          name: 'frontVariant'
+        }
+      },
+      args,
+      constants: [
+        {
+          type: 'template',
+          name: 'style',
+          value: '<div style="margin-bottom: 40px; width: 300px">%TEMPLATE%</div>'
+        }
+      ]
+    })
+};
+
+/**
+ * The sd-flipcard can be displayed in 3:4 or 16:9 aspect ratio.
+ */
+
+export const Ratio = {
+  parameters: { controls: { exclude: ['ratio'] } },
+  render: (args: any) =>
+    generateTemplate({
+      axis: {
+        x: {
+          type: 'attribute',
+          name: 'ratio'
+        }
+      },
+      args
+    }),
+  decorators: [
+    (story: any) =>
+      html`<style>
+          td.template {
+            width: 50%;
+          }
+        </style>
+        ${story()}`
+  ] as unknown
+};
+
+/**
+ * Use the `front`, `back`, `mediaFront` and `mediaBack` slots to add content to the flipcard.
+ */
+export const Slots = {
+  parameters: {
+    controls: { exclude: ['front', 'back', 'mediaFront', 'mediaBack'] }
+  },
+  render: (args: any) => {
+    return html`
+      ${['front', 'back', 'mediaFront', 'mediaBack'].map(slot => {
+        return generateTemplate({
+          axis: {
+            x: {
+              type: 'slot',
+              name: slot,
+              title: 'slot=..',
+              values: [
+                {
+                  value: `<div slot='${slot}' class="slot slot--border slot--background w-full h-full"></div>`,
+                  title: slot
+                }
+              ]
+            }
+          },
+          args,
+          constants: [
+            {
+              type: 'template',
+              name: 'style',
+              value: '<div style="margin-bottom: 40px; width: 300px">%TEMPLATE%</div>'
+            },
+            {
+              type: 'attribute',
+              name: 'frontVariant',
+              value: 'gradient-dark-top'
+            },
+            {
+              type: 'attribute',
+              name: 'backVariant',
+              value: 'gradient-dark-bottom'
+            }
+          ]
+        });
+      })}
+    `;
+  }
+};
+
+/**
+ * Use the `base`, `front`, `back`, `frontSlotContainer`, `backSlotContainer`, `mediaFront`, `mediaBack`, `frontSecondaryGradient` and `backSecondaryGradient` parts to style the flipcard.
+ */
+export const Parts = {
```

Comment:
the `mediaBack` part is not visible as the sample only shows an image on the front.